beacon-chain/forkchoice/protoarray/types.go

```diff
@@ -44,15 +44,17 @@ type Node struct {
 	weight         uint64                       // weight of this node.
 	bestChild      uint64                       // bestChild index of this node.
 	bestDescendant uint64                       // bestDescendant of this node.
-	graffiti       [fieldparams.RootLength]byte // graffiti of the block node.
+	optimistic     OptimisticStatus             // optimistic status of this node
```

Comment:
```suggestion
	status     status             // status of this node
```

Reply:
I think status is too generic, perhaps a different name?